pkgs/development/libraries/reproc/default.nix

```
@@ -12,6 +12,19 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-LWzBeKhE7cSiZsK8xWzoTdrOcPiU/zEkmi40WiFytic=";
   };
 
+  patches = [
+    (fetchpatch{
+      name = "gcc-13-2";
+      url = "https://github.com/DaanDeMeyer/reproc/commit/0b23d88894ccedde04537fa23ea55cb2f8365342.patch";
+      sha256 = "sha256-QyC0UcKAWCKSvSvyZTLI2eF/TuuqbGGH6cOQrS2DiCE=";
+    })
+    (fetchpatch{
+      name = "gcc-13-1";
+      url = "https://github.com/DaanDeMeyer/reproc/commit/9f399675b821e175f85ac3ee6e3fd2e6056573eb.patch";
+      sha256 = "sha256-h/gnDFPWPpUFkys10YXjjEPibgRT1atHSVwbO0kId+U=";
+    })
```

```suggestion
    # following two patches fix GCC 13 build
    (fetchpatch {
      name = "reproc-gcc-13-1.patch";
      url = "https://github.com/DaanDeMeyer/reproc/commit/0b23d88894ccedde04537fa23ea55cb2f8365342.patch";
      hash = "sha256-QyC0UcKAWCKSvSvyZTLI2eF/TuuqbGGH6cOQrS2DiCE=";
    })
    (fetchpatch {
      name = "reproc-gcc-13-2.patch";
      url = "https://github.com/DaanDeMeyer/reproc/commit/9f399675b821e175f85ac3ee6e3fd2e6056573eb.patch";
      hash = "sha256-h/gnDFPWPpUFkys10YXjjEPibgRT1atHSVwbO0kId+U=";
    })
```

Can you make sure the patch files have a better derivation name in the nix store (that ideally includes "reproc") and continue to end in `.patch`?

Otherwise this PR looks great, thank you!